parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -45,15 +45,14 @@ installDeps() {
     fi
 
     pkg_list+=(blobfuse2=${BLOBFUSE2_VERSION})
-    if [[ $(isARM64) != 1 ]]; then
-        # blobfuse2 is installed for all ubuntu versions, it is included in pkg_list
-        # for 22.04, fuse3 is installed. for all others, fuse is installed
-        # for 16.04, installed blobfuse1.3.7, for all others except 22.04, installed blobfuse1.4.5
-        if [[ "${OSVERSION}" == "22.04" ]]; then
-            pkg_list+=(fuse3)
-        else
-            pkg_list+=(blobfuse=${BLOBFUSE_VERSION} fuse)
-        fi
+
+    # blobfuse2 is installed for all ubuntu versions, it is included in pkg_list
+    # for 22.04, fuse3 is installed. for all others, fuse is installed
+    # for 16.04, installed blobfuse1.3.7, for all others except 22.04, installed blobfuse1.4.5
+    if [[ "${OSVERSION}" == "22.04" ]]; then
+        pkg_list+=(fuse3)
```

Comment:
PR description "Removing FIPS guard for fuse3 installation, because FIPS has the package now." is not accurate. 'FIPS' should be 'ARM64'. 

Long time ago ARM64 doesn't have blobfus2 and fuse3, now it seems that arm64 has both now.

Reply:
Sorry about that 😅 I updated the description